### ISSUE TITLE
dev: Add Advanced tab in Settings

### ISSUE THREAD

**Issue by Jerry23011** [tisfeng/Easydict#350]
Since the General tab is fully packed, we can create a new Advanced tab and move some sections there.

SF Symbol: `gearshape.2`

**Comment by tisfeng** [comment]
Agreed, the current General Tab config options are complex, can be confusing to new users, we should simplify it.

**Comment by tisfeng** [comment]
A previous user email feedback said that the settings options can be a bit of a pain to set up, and I think this might needs to be improved.

Splitting up the General Tab to keep it simple and moving some of the options to the Advanced Tab could be a way forward.

If there are any ideas on how to improve it and which options to move, please feel free to RP.

<img width="755" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/9592650a-b5d4-4d88-a75e-e4c982cb015c">

**Comment by tisfeng** [comment]
@Jerry23011  I have added a Advanced tab, can you help move some of options from General to Advanced tab?

<img width="686" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/a27bfca2-5c8d-472e-b2d4-bedc1049caf7">

**Comment by Jerry23011** [comment]
I'll take a look
